rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ObjectController.java

```diff
@@ -140,13 +130,24 @@
 
         final Stream<RouterCertificate> combinedAssertions = Stream.concat(filteredRouterCertificates, bgpSecAssertions).distinct();
 
+        Pair<Boolean, Long> allDoneForTa = Time.timed(() -> storage.readTx(tx ->
```

Comment:
Why is it timed? Do we want to add a metric for this?

Reply:
Removed, it was just for the debugging and slipped into the commit.